network/yarn/pom.xml

```diff
@@ -54,6 +54,11 @@
       <groupId>org.apache.hadoop</groupId>
       <artifactId>hadoop-client</artifactId>
     </dependency>
+    <dependency>
```

Comment:
note: this is unrelated but sbt started complaining about this dependency while testing this change.